src/Intl/Transliterator/Transliterator.php

```diff
@@ -0,0 +1,676 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Polyfill\Intl\Transliterator;
+
+/**
+ * Partial intl implementation in pure PHP.
+ *
+ * Implemented:
```

Comment:
this phpdoc seems incomplete

Reply:
for now I added only "create" and "transliterate" here